bundles/org.openhab.binding.hue/src/main/java/org/openhab/binding/hue/internal/api/dto/clip2/helper/Setters.java

```diff
@@ -52,6 +57,8 @@
 @NonNullByDefault
 public class Setters {
 
+    private static final Set<ResourceType> LIGHT_TYPES = Set.of(ResourceType.LIGHT, ResourceType.GROUPED_LIGHT);
```

Comment:
See my other discussion about grouped_light -- is it proven that grouped_light has a problem? Why would you need to fix something if it is not proven to be broken? The two API's are different, as shown below, so there are at least some grounds to suspect that the bridge would handle them different. Specifically grouped_light  does not use ColorXy or ColorTemperature..

- https://developers.meethue.com/develop/hue-api-v2/api-reference/#resource_light_get
- https://developers.meethue.com/develop/hue-api-v2/api-reference/#resource_grouped_light_get

Reply:
The issue with the **light** resource was also not "proven" by you - I'm the only one of us having experienced the issue, which is specifically caused by having **on** in one resource and **dimming** in another. Both are also part of **grouped_light**. So I see no reason **not to fix** this.

I could probably reproduce it by creating a zone for my dimmer having the issue already, but in any case we don't want this coming back to us in a future bug, even if it currently seems theoretical with the current firmware and the particular devices and configurations we are using for testing. Especially because it required no additional effort or complexity to fix this as well.

Reply:
The other way around.

If **grouped_light** has the same issue, which logically would make sense since it can also contain **on**/**dimming** values, we should rather include the fix for this scenario as well.

On the other hand, if **grouped_light** doesn't have the same issue, the fix will do no harm. In fact, it won't even have any impact, since nothing would be merged.

I really don't see how this could create a bug that wouldn't also be created for **light** then.

Reply:
Ok. Closing this issue.